### PR TITLE
az-acr: add page 

### PR DESCRIPTION
**Pull request by BS14**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

For: https://github.com/tldr-pages/tldr/issues/6141

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
@BS14 Welcome to tldr and thanks for your contribution. Can you sign the CLA for this PR to go through?

**Comment by BS14** [comment]
@kbdharun I have already signed CLA. Not sure why it is not being reflected in the checks. Any idea on this ? 

image
image

**Comment by kbdharun** [comment]
> @kbdharun I have already signed CLA. Not sure why it is not being reflected in the checks. Any idea on this ?

Screenshot from 2023-10-10 10-47-50

I can confirm you have signed the CLA (checked it in the admin interface). But seems like the commits you made don't have the same email address as your GitHub account/your email isn't configured locally. Feel free to try out these steps https://github.com/tldr-pages/tldr/blob/main/contributing-guides/git-terminal.md#changing-the-email-of-any-commits and also set the name and email globally using:

```
git config --global user.name "Your name"
git config --global user.email "your@email.com"
```

File: pages/common/az-acr.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker pull {{registry_name}}.azurecr.io/{{image_name}}:{{tag}}`
+
+- Delete an image from a registry:
+
+`az acr repository delete --name {{registry_name}} --repository {{image_name}}:{{tag}}`
+
+- Delete a managed container registry:
+
+`az acr delete -name {{registry_name}} --resource-group {{resource_group}} --yes`
```
Comment (kbdharun) [inline pages/common/az-acr.md:32]:
```suggestion
`az acr delete --name {{registry_name}} --resource-group {{resource_group}} --yes`
```

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**